doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -498,6 +498,14 @@ Other libraries
 
   * The library now supports using the GPIO expander for the buttons, switches, and LEDs on the nRF9160 DK.
 
+* :ref:`app_event_manager` library:
+
+  * Added :c:macro:`APP_EVENT_ID` macro.
+
+* :ref:`event_manager_proxy` library:
+
+  * :ref: :c:func:`event_manager_proxy_subscribe` redundant ``remote_event_name`` argument removed.
```

Comment:
I think the library docs also needs to be updated. https://res.developer.nordicsemi.com/ncs/doc/latest/nrf/libraries/others/event_manager_proxy.html#subscribing-to-remote-events
```suggestion
  * The :c:func:`event_manager_proxy_subscribe` function is deprecated and the ``remote_event_name`` argument used by this function is removed.
```
Considering :c:func:`event_manager_proxy_subscribe` function  is now deprecated.

Reply:
I would not say that the function is deprecated....

Reply:
The documentation part updated.